readthedocs/doc_builder/templates/doc_builder/data.js.tmpl

```diff
@@ -1,10 +1,13 @@
-var READTHEDOCS_DATA = {{ data_json|safe }}
+{% load core_tags %}
+
+
+var READTHEDOCS_DATA = {{ readthedocs_data|escapejson:4 }}
 
 // Old variables
-var doc_version = "{{ current_version }}";
-var doc_slug = "{{ slug }}";
-var page_name = "{{ pagename }}";
-var html_theme = "{{ html_theme }}";
+var doc_version = "{{ current_version|escapejs }}";
+var doc_slug = "{{ slug|escapejs }}";
+var page_name = "{{ pagename|escapejs }}";
+var html_theme = "{{ html_theme|escapejs }}";
```

Comment:
I think `"` are inserted by the filter.

Reply:
Reading the django code, looks like they don't include `"`

Reply:
I'm not sure where you're looking but it is included as far as I can tell.

    >>> from django.template.defaultfilters import escapejs_filter
    >>> escapejs_filter('"')
    '\\u0022'

https://github.com/django/django/blob/1.11/django/utils/html.py#L54-L76

Reply:
I mean the `"` surrounding the variable. In jinja you just need to put `var foo = {{ bar|tojson }}`. Django needs  `var foo = "{{ bar|escapejs }"`

Reply:
Got it. Correct. That's different.